### ISSUE TITLE
tests fail after upgrade to `0.6.0` due to PocketIC binary taking too long to start

### ISSUE THREAD

**Issue by letmejustputthishere** [hadronous/pic-js#33]
```
Error: The PocketIC binary took too long to start. Please try again.
 ❯ node_modules/@hadronous/pic/src/pocket-ic-server.ts:65:13
 ❯ Timeout.runPoll node_modules/@hadronous/pic/src/util/poll.ts:22:24
```

you can find the code [here](https://github.com/letmejustputthishere/vite-sveltekit-motoko-ii/tree/feature/upgrade-deps).
simply run 
- `dfx start --clean --background`
- `npm run setup`
- `npm run test`

**Comment by jake-beardo** [comment]
I am also getting this error

**Comment by nathanosdev** [comment]
Thanks for reporting. Are you also on MacOS @jake-beardo?

**Comment by jake-beardo** [comment]
Hi @nathanosdev yes i am on MacOS, I will try peg to v0.5.0 and see if it changes anything

**Comment by letmejustputthishere** [comment]
`0.5.0` worked for me on macOS

**Comment by jake-beardo** [comment]
I managed to run 1 test sucessfully using bun test

but every test I run afterwards seem to fail because of this error 

```
bun test                                                                                                                                         01:09:54 PM
bun test v1.0.30 (1424a196)

src/orderbook/test/integration/main.test.ts:
thread 'tokio-runtime-worker' panicked at library/std/src/io/stdio.rs:1021:9:
failed printing to stdout: Broken pipe (os error 32)
thread 'tokio-runtime-worker' panicked at rs/pocket_ic_server/src/state_api/state.rs:495:45:
join failed!: JoinError::Panic(Id(49), ...)
ConnectionClosed: The socket connection was closed unexpectedly. For more information, pass `verbose: true` in the second argument to fetch()
 path: "http://127.0.0.1:50158/instances/7/update/execute_ingress_message"
✗ OrderBook > should start at [47.42ms]
✗ OrderBook > should start at

 0 pass
 2 fail
Ran 2 tests across 1 files. [123.00ms]
```

**Comment by nathanosdev** [comment]
@jake-beardo this looks like a separate issue, could you please open a new issue with more details? What version of pic-js are you running and share a stripped down example of a test that's failing.

**Comment by nathanosdev** [comment]
The original reported issue has been fixed and published in v0.6.1.
